### DISCUSSION TITLE
Hot Topics for InnerSource Commons Community Calls

### DISCUSSION THREAD

**Discussion by claredillon** [InnerSourceCommons/InnerSourceMarketing#133] [Ideas]
The following are some of the topics of interest, and notes about format ideas, that have been gathered from our Community Call attendees. We are open to experimenting with new topics and new ways to format the call.

Please add in your own ideas on:
- what themes / topics are most interesting to you
- how we might experiment with the community call format.

Please also feel free to volunteer to speak, or even request a particular speaker. 

## Hot Topics
- InnerSource Communications & Story Telling - best practices and other internal marketing practices
- Enabling InnerSource Collaborations 
  - kick-starting and scaling collaborations
  - Addressing from org/system level and psychological level (e.g. building trust)
-   InnerSource Failures - pitfalls to avoid
- Technical Deep Dives - e.g. enabling security in InnerSource Collaborations
- Legal & Compliance Priorities and how to address them
- Findability, visibility, discoverability for InnerSource projects and experienced people
- InnerSource metrics
- InnerSource ROI
- Strategies for dealing with middle management
- InnerSource 101 - getting started
- InnerSource Year 2 - you've had your initial success, what's next?

## Community Call Format Ideas
- Pattern Sprints (to end up with an artefact from the call)
- Hackathon e.g. to implement metrics or integrate tools for a fake org in GitHub
- Introduce more panel sessions
- Mix presentations from InnerSource practitioners and perhaps experts in related areas (e.g. comms experts)

**Comment by dellagustin-sap** [comment]
I'd like tu suggest adding Secure InnerSource to the Hot Topics, there is a lot of people interested in this topic.
References:
https://github.com/InnerSourceCommons/InnerSourcePatterns/issues/369
https://innersourcecommons.slack.com/archives/C2EFRTS6A/p1637788368015300

**Reply by claredillon** [reply]
Great suggestion @dellagustin-sap - would you be interested in being part of the discussion? :)

**Comment by claredillon** [comment]
What do people think about a session on the topic of "Which project types work best with InnerSource?". We've heard anecdotally that it is very useful when folks are doing DevOps, or cloud transformations (and need new common tooling). 

We've also had some discussions about the optimal project set-up wrt to commitments re ongoing support etc. 

Would this be an interesting topic for a community call to explore more deeply?

**Comment by clcoffey** [comment]
I would like to suggest the topic of Innersource Programme Offices  - how to establish one and what problems can be addressed through it. 

Inspired by https://github.com/InnerSourceCommons/InnerSourcePatterns/issues/365

**Comment by claredillon** [comment]
Notes from today's Lean Coffee Community Call - includes feedback on topics we might cover from September...

**InnerSource local communities in local language**

- Challenge in over-coming cultural hesitancy to share
- Good to find an internal champion
- Internal community of practice events with curated external speakers can work.
- Case studies can be a path for getting stories out (easier to get approval than for speakers, for example)
- Can de-escalate fears/hesitancy by showing wins
- Strategic humility works well (plan to experiment, not make upfront claims of success)

**Security**

- Discussed what tooling/configuration to use for various security requirements
- AGREED: A topic for further discussion in the community calls; ideas include getting Security professional to speak (rather than developer perspective). Another idea is to get tools vendors to share configurations in place in various orgs to respond to specific security concerns (e.g. making README viewable, but code only available if contributor meets certain criteria/governance rules)

**Toolchains, portals, discoverability & Getting kick-started**

- SAP portal is “starter” portal. https://github.com/SAP/project-portal-for-innersource
- Alternative portal/dev experiences can be based on backstage/Spotify (asset management for software projects, not specifically for InnerSource)
- AGREED: Delve into this topic in future community calls; e.g. discoverability in large complex orgs; how to facilitate InnerSource when teams are working with different tooling.
-
